app/templates/components/ui-kit/link.hbs

```diff
@@ -0,0 +1,12 @@
+{{#let (element this.tag) as |Tag|}}
+  <Tag
+    class={{this.allClasses}}
+    href={{this.href}}
```

Comment:
`href` makes sense only for `<a>` tag. I think, we shouldn't expose `@tag` attribute here